tests/subsys/net/openthread/rpc/common/test_rpc_env.h

```diff
@@ -34,6 +34,8 @@
 #define RPC_RSP(...)      RPC_PKT(0x01, 0xff, 0x00, 0x00, 0x00 __VA_OPT__(,) __VA_ARGS__, 0xf6)
 #define NO_RSP            RPC_PKT()
 
+#define CBOR_LIST(...) 0x9F, __VA_ARGS__, 0xFF
```

Comment:
Could you move this to "/* Macros for constructing CBOR data items. */" section above?